solutions/swb-reference/src/index.ts

```diff
@@ -4,10 +4,21 @@
  */
 
 /* eslint-disable no-new */
+import { WorkbenchAppRegistry } from '@aws/workbench-core-infrastructure';
 import * as cdk from 'aws-cdk-lib';
+import { config } from 'dotenv';
```

Comment:
i'm glad to see we're using `dotenv` but why do we need it now? we've been able to read env files before right?

Reply:
I saw the example of the test application and followed their pattern, I didn't want to have it in yaml as those values are normally modified by users and this values should not be touched by users

Reply:
On a second tought I think this values can be on constants while we figure a way to retrieve them dynamically